src/mocks/server_request.ts

```diff
@@ -1,11 +1,25 @@
 import { BufReader, ServerRequest } from "../../deps.ts";
 
+export interface MockServerRequestOptions {
+  headers: { [key: string]: string };
+  body?: Deno.Buffer;
+  server?: any; // TODO what is server type
```

Comment:
Dont know its purpose, what type it should be?

Reply:
@crookse What is the server for?

Reply:
pretty sure that's old code. in drash, the `Drash.Http.Server` object used to get attached to the drash's `request` object so that developers could do:

```
request.server.doSomething()
```

but i must've found that it wasn't needed and took out all that logic. you can remove this. i checked the drash codebase and we're not using it. also, for context, the `ServerRequest` mock object that you're working with came from drash's unit tests. it was catered strictly for drash, but now that it's in rhum, it's been generalized to fit needs of other projects. it may be the case that i just forgot to remove the `server` option since we developed rhum at a quick pace. probably too quick of a pace.

Reply:
removed `server` option property